doc/migration_guide.rst

```diff
@@ -7,6 +7,16 @@ This guide attempts to be, but is not, complete. If you run into a problem while
 converting code that does not seem to be described here, please open an issue on
 Github.
 
+Documentation
+-------------
+
+Most of the API references are now powered by `Breathe
+<https://github.com/breathe-doc/breathe>`_. Without Breathe the documentation
+will lack almost all API details and will be much less helpful. Additionally,
+the Sphinx extension `togglebutton
+<https://pypi.org/project/sphinx-togglebutton/>`_ must be installed to
+successfully build the documentation.
```

Comment:
Woah - is this really correct? We *must* have this extension, to build the docs at all? I don't think that can work.

Per https://repology.org/project/python:sphinx-togglebutton/versions effectively zero Linux distributions include this package, which would mean that everyone installing the Linux packages from their distro would get no (or broken) documentation.

Reply:
Mhm, indeed. Its likely possible to ship this (or a subset of this) extension so that we don't need to pull it out of `pip` or similar.